src/network/networkprotocol.h

```diff
@@ -212,7 +212,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #define PASSWORD_SIZE 28       // Maximum password length. Allows for
                                // base64-encoded SHA-1 (27+\0).
 
-#define FORMSPEC_API_VERSION 1
+#define FORMSPEC_API_VERSION 2
```

Comment:
we never bump if it's an attribute addition if i'm correct

Reply:
This PR is a copy of #5395, don't worry i'll revert this change.